src/app/core/views/portinfo/digital/index.ts

```diff
@@ -45,4 +45,26 @@ export const DigitalPortInfo: PortInfoRecord<DigitalComponent> = {
             "outputs": [CalcPortPos(V(0.5, 0), V(1, 0))], // 1 output
         })),
     },
+    "ORGate": {
+        ...DefaultDigitalPortInfo,
+        AllowChanges: true,
+        ChangeGroup:  "inputs",
+
+        // Generate configs for 2->8 input ports
+        PositionConfigs: [2,3,4,5,6,7,8].map((numInputs) => ({
+            "inputs":  CalcQuadCurvePortPositions(numInputs, 1.2, .48, V(-1, 0)),
+            "outputs": [CalcPortPos(V(0.58, 0), V(1, 0))], // 1 output
+        })),
+    },
+    "NORGate": {
+        ...DefaultDigitalPortInfo,
+        AllowChanges: true,
+        ChangeGroup:  "inputs",
+
+        // Generate configs for 2->8 input ports
+        PositionConfigs: [2,3,4,5,6,7,8].map((numInputs) => ({
+            "inputs":  CalcQuadCurvePortPositions(numInputs, 1.2, .48, V(-1, 0)),
+            "outputs": [CalcPortPos(V(0.8, 0), V(.7, 0))], // 1 output
```

Comment:
I think the outputs here can be the same as `ORGate`, the circle for the not can simply lay on-top of the port
![Screen Shot 2022-11-08 at 2 36 31 PM](https://user-images.githubusercontent.com/29997057/200658745-3a0c1663-16db-4711-b37f-29339a5179ab.png)
(this is a screenshot from master)